torch/_torch_docs.py

```diff
@@ -1378,6 +1378,25 @@ def merge_dicts(*dicts):
             [-1.2329,  1.9883,  1.0551]])
 """.format(**common_args))
 
+add_docstr(torch.logcumsumexp,
+           r"""
+logcumsumexp(input, dim, out=None) -> Tensor
+Returns the logarithm of the cumulative summation of the exponentiation of
+elements of :attr:`input` in the dimension :attr:`dim`. 
+
+.. math::
+    \text{{logsumexp}}(x)_{{i}} = \log \sum_j \exp(x_{{ij}})
```

Comment:
I'm not an expert on our convention here, but this doesn't seem correct.

For one, this looks like a reduction, but this function isn't a reduction.

I would have expected something roughly like:
` \text{{logsumexp}}(x)_{{ij}} = \log \sum_{k<=j} \exp(x_{{ik}})`

Reply:
I think following the notation in cumsumexp documentation, we should write:

```
For summation index :math:`j` given by `dim` and other indices :math:`i`, the result is

    .. math::
        \text{{logcumsumexp}}(x)_{{i}} = \log \cumsum_j \exp(x_{{ij}})
```